### PR TITLE
recently viewed products

### PR DESCRIPTION
**Pull request by phamtin**

### CODE REVIEW

**Comment by phamtin** [comment]
ready

File: src/pages/product-detail/elements/RecentProduct.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    id: product.id,
+                    price: product.price,
+                    heading: product.heading,
+                    linkImage: product.linkImage,
+                    description: product.description
+                }
+            };
+            return (
+                <div className="row mb-4">
+                    {console.log(product)}
```
Comment (phamvanbk11) [inline src/pages/product-detail/elements/RecentProduct.js]: Xóa log đi em

File: src/pages/product-detail/elements/ListRecentlyProduct.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        for (let i = 0; i < localStorage.length; i++) {
+            if (localStorage.key(i).substring(0, 14) === "viewedProduct-") {
+                const key = localStorage.key(i);
+                listKey.push(key);
+            }
+        }
+        for (let i = 0; i < listKey.length; i++) {
+            listProduct.push(localStorage.getItem(listKey[i]));
+        }
+        console.log(listProduct);
```
Comment (phamvanbk11) [inline src/pages/product-detail/elements/ListRecentlyProduct.js]: Bỏ log đi nhé

**Comment by phamvanbk11** [comment]
commented

**Comment by phamtin** [comment]
ready

**Pull request merged**